### PR TITLE
You can see who created the aspect in its dashboard

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

**Comment by Comoque1** [comment]
Probably should move First name/Last name above each other, when the text is longer.
image

File: src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx [hunk 1]
```diff
@@ -102,6 +96,15 @@ const AspectDashboardContainer: FC<AspectDashboardContainerProps> = ({
...
   const error = hubError ?? challengeError ?? opportunityError;
 
+  const { data: creatorData, loading: loadingCreator } = useAspectCreatorQuery({
+    variables: { userId: aspect?.createdBy ?? '' },
+    skip: !aspect,
+  });
+  const creator = creatorData?.user;
+  const creatorAvatar = creator?.profile?.avatar?.uri;
+  const creatorName = creator?.displayName;
+  const createdDate = aspect && new Date(aspect.createdDate);
```
Comment (me-andre) [inline src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx:106]: I would avoid to pass rich objects as props if they could be instantiated in the consuming view. If there's absolute need for that, you have to wrap its instantiation in `useMemo()`.

File: src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx [hunk 2]
```diff
@@ -125,11 +128,8 @@ const AspectDashboardContainer: FC<AspectDashboardContainerProps> = ({
     [messages, user, isAuthenticated]
   );
 
-  const permissions: AspectDashboardPermissions = {
-    canReadComments: commentsPrivileges.includes(AuthorizationPrivilege.Read) ?? false,
-    canPostComments: commentsPrivileges.includes(AuthorizationPrivilege.CreateComment) ?? false,
-    canDeleteMessage,
-  };
+  const canReadComments = commentsPrivileges.includes(AuthorizationPrivilege.Read) ?? false;
```
Comment (me-andre) [inline src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx:131]: `includes()` always returns a `boolean`.

**Review by me-andre — CHANGES_REQUESTED** [review]

**Review by me-andre — APPROVED** [review]

**Comment by hero101** [comment]
Closing this because the license does not pass.

**Pull request closed without merging**